### PR TITLE
61 center button texts

### LINKED ISSUE #61: ffe-buttons: Text is aligned left in block level buttons
The text in block level buttons is aligned to the left, should be center. This bug is present despite both `text-align: center;` and `align-items: center;` (on the label element) being set.

screenshot from 2018-03-07 15-19-09

### PR DESCRIPTION
**Pull request by selbekk**
This pull request fixes a bug with ffe-buttons that doesn't center the text.

In addition, this pull request fixes an issue in the button docs that stopped us from adding a ghost modifier example.

Fixes #61

### CODE REVIEW

File: packages/ffe-buttons-react/src/ActionButton.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ ActionButton.propTypes = {
     disabled: bool,
     /** The rendered element, like an `<a />` or `<Link />` */
     element: oneOfType([func, string]),
+    /** Applies the ghost modifier if true. */
+    ghost: bool,
```
Comment (wkillerud) [inline packages/ffe-buttons-react/src/ActionButton.js:39]: This change isn't really related to `docs`, so I think it should either be skipped or done in its own commit.

**Review by wkillerud — CHANGES_REQUESTED** [review]

File: packages/ffe-buttons-react/src/ActionButton.js (hunk 1 above)
Comment (selbekk) [reply packages/ffe-buttons-react/src/ActionButton.js:39]: It kind of is related though, since it changes the order this prop shows up in the docs.

Comment (wkillerud) [reply packages/ffe-buttons-react/src/ActionButton.js:39]: That's a stretch, but I'll allow it :wink:

**Review by wkillerud — APPROVED** [review]

**Review by antidecaf — APPROVED** [review]
